containerWithMostWater.js

```diff
@@ -0,0 +1,23 @@
+// This file contains solution to the following problem:
+// https://leetcode.com/problems/container-with-most-water/
+
+const maxArea = (height) => {
+  let leftPointer = 0;
+  let rightPointer = height.length - 1;
+  let sum = Math.min.apply(null, height);
+
+  while (leftPointer < rightPointer) {
```

Comment:
`ptr` is a short version of `pointer`, better use short names

Reply:
Clear